### PR TITLE
changed string concatenations to f-strings, fixed two typos

### PR DESCRIPTION
**Pull request by RyanKimbrell**
#41 I changed most of the string concatenations to f-strings. I did leave all of the concatenations involving ANSI escape codes as they originally were because I could not find out how the f-string conversion might affect them. 

Lastly, I fixed two typos in the markdown documentation:
"aks.py" -> "ask.py" in the readme
"signle quotes" -> "single quotes" in the contributing

### CODE REVIEW

File: ask.py [hunk 1]
```diff
@@ -25,8 +25,8 @@ def get_ask_config(source_root):
 	if source_root:
 		source_root += '/'
 
-	if os.path.isfile(source_root + 'Askfile'):
-		with open(source_root + 'Askfile', 'r') as f:
+	if os.path.isfile(f'{source_root}Askfile':
```
Comment (Buscedv) [inline ask.py:28]: Missing closing parenthesis

**Review by Buscedv — CHANGES_REQUESTED** [review]
Thank you for your contribution 🚀!

There was one small syntax error on line *28*, but other than that, perfect 👍!

I did some testing myself with f-strings and 033 syntax, and it doesn't seem to work, so no worries there.

**Comment by RyanKimbrell** [comment]
Added the closed parenthesis at line 28! Thank you 👍

**Review by Buscedv — APPROVED** [review]
Awesome work!👍 Thank you so much for your contribution!! ✔️

**Pull request merged**